### PR TITLE
hotfix for large file sizes

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
Please see [here](https://github.com/raft-tech/TANF-app/pull/2532), missed one config.

### CODE REVIEW

**Review by raftmsohani — APPROVED** [review]
LGTM!

**Review by jtimpe — APPROVED** [review]

**Comment by andrew-jameson** [comment]
Nginx blocking filenames thinking they're extensions

File: scripts/zap-scanner.sh [hunk 1]
```diff
@@ -65,6 +65,9 @@ ZAP_CLI_OPTIONS="\
   -config globalexcludeurl.url_list.url\(2\).regex='^https?://.*\.amazonaws\.(?:com|org|net)/.*$' \
   -config globalexcludeurl.url_list.url\(2\).description='TDP S3 buckets' \
   -config globalexcludeurl.url_list.url\(2\).enabled=true \
```
Comment (raftmsohani) [inline scripts/zap-scanner.sh:67]: filtering login.gov and identity sandbox from the scan. We shouldn't need to scan the external sites

Comment (raftmsohani) [reply scripts/zap-scanner.sh:67]: Screenshot 2023-05-31 at 1 12 57 PM

Comment (ADPennington) [reply scripts/zap-scanner.sh:67]: @raftmsohani does this last fix address all the issues flagged [here](https://app.circleci.com/pipelines/github/raft-tech/TANF-app/13902/workflows/3dc39e60-a784-4a5f-8a07-4a40aabdb0aa/jobs/41590/artifacts)?

i thought we resolved the sql injection?

Comment (raftmsohani) [reply scripts/zap-scanner.sh:67]: > @raftmsohani does this last fix address all the issues flagged [here](https://app.circleci.com/pipelines/github/raft-tech/TANF-app/13902/workflows/3dc39e60-a784-4a5f-8a07-4a40aabdb0aa/jobs/41590/artifacts)?
> 
> i thought we resolved the sql injection?

I also thought we have changed: https://github.com/raft-tech/TANF-app/blob/238e85e6cf778e845b56641a62c059ea50236d3e/tdrs-frontend/reports/zap.conf#L82 to be ignored.

Comment (raftmsohani) [reply scripts/zap-scanner.sh:67]: So we will need to change this line to "IGNORE"

**Review by ADPennington — COMMENTED** [review]
@raftmsohani not sure if there should be another ticket to track work, but do you happen to know why the qasp-label owasp scans continue to scan non-tdp urls? https://app.circleci.com/pipelines/github/raft-tech/TANF-app/13919/workflows/01b69a38-3896-4372-8e24-d0858ec5b480/jobs/41679/artifacts

**Review by ADPennington — APPROVED** [review]
👍🏾 will investigate qasp label scan in #2563

**Pull request merged**